README.md

```diff
@@ -147,6 +148,73 @@ res, err := goreq.Request{
 }.Do()
 ```
 
+
+### Tags
```

Comment:
The GET section (https://github.com/evalphobia/goreq/blob/feature/squash-embed-struct/README.md#get) already mentions about `url` tags. Let's include this section after that one instead of placing it after the POST section as it is today.